GoogleDataTransport/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# v7.0.1
+- `GDTCORFlatFileStorage`: keep not expired events when expired batch removed. (#6010)
```

Comment:
Move into 7.0.0 section.

Reply:
I'll remove it entirely because it is not too separate from the refactoring mentioned in 7.0.0